src/assets/scripts/client/aircraft/StripView/StripViewController.js

```diff
@@ -257,7 +277,53 @@ export default class StripViewController {
      * @private
      */
     // eslint-disable-next-line no-unused-vars
-    _onStripListClickOutsideStripViewModel = (event) => {
-        this.findAndDeselectActiveStripView();
-    };
+    _onStripListClickOutsideStripViewModel = (event) => this.findAndDeselectActiveStripView();
+
+    /**
+     * Generate a unique number to represent a `CID`
+     *
+     * Should be displayed with leading zeros, so a `CID` value of `1` should be displayed as `001`
+     *
+     * @for StripViewController
+     * @method _generateCidNumber
+     * @return nextCid {number}
+     * @private
+     */
+    _generateCidNumber() {
+        const nextCid = _random(1, CID_UPPER_BOUND);
+        const nextCidLeft = nextCid - 1;
+        const nextCidRight = nextCid + 1;
+
+        if (
+            this._cidNumbersInUse.indexOf(nextCid) !== -1 ||
+            this._cidNumbersInUse.indexOf(nextCidLeft) !== -1 ||
+            this._cidNumbersInUse.indexOf(nextCidRight) !== -1
+        ) {
+            this._generateCidNumber();
```

Comment:
:man_facepalming: I see what happened. Sorry, I was definitely not real clear. What I meant by non-sequential was not to prevent the _existence_ of "adjacent" numbers, but rather to ensure that they are _generated non-sequentially_. As in, the first 3 CIDs ought not to be `001, 002, 003`, but that's not to say that we couldn't have `[054, 126, 354, 355, 617]`. My B!